docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -286,6 +297,31 @@ For example, the following command runs tests from the
 testcafe all tests/sample-fixture.js -S -s screenshots
 ```
 
+### -p, --screenshot-path-pattern
+
+Specifies a custom pattern to compose screenshot files' relative path and name. This pattern ovverrides the default [path pattern](#path-patterns).
+
+You can use the following placeholders in the pattern:
+
+Placeholder | Description
+----------- | -----------
+`${DATE}` | The test run's start date (YYYY-MM-DD).
+`${TIME}` | The test run's start time (HH-mm-ss).
+`${TEST_INDEX}` | The test's index.
+`${FILE_INDEX}` | Numbers a file in NNN format for files of the same name in the same folder.
```

Comment:
I don't know that is the `NNN format`.
I propose to rename as 'The screenshot file index`.